### PR TITLE
Add Slides model & seed initial slides.

### PR DESCRIPTION
**Pull request by horrorsaur**
This is the groundwork for the How to play "instructable" system we are going for. 

This adds a Slides model which can accept 3 fields:
* title
* description
* img_url

Using the data contained in the db, we can decide how to show these to the user.

**_Note:_** I added to the seed file (sorry). You will need to run **rake db:seed** in order to get some data

### CODE REVIEW

File: app/controllers/slides_controller.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    # Use callbacks to share common setup or constraints between actions.
+    def set_slide
+      @slide = Slide.find(params[:id])
+    end
+
+    # Only allow a trusted parameter "white list" through.
+    def slide_params
+      params.require(:slide).permit(:title, :description, :img_url)
+    end
+end
```
Comment (horrorsaur) [inline app/controllers/slides_controller.rb:58]: I left the boilerplate code generated by Rails so we can make some kind of CMS for this in a future update. I commented out the usage of 'resources' in the routes. Is that enough to ensure this stuff is never reached?

Comment (horrorsaur) [reply app/controllers/slides_controller.rb:58]: Also - 
I will have to update this again with the final slides context from Elyse/Danielle and reseed the slides

Comment (efatsi) [reply app/controllers/slides_controller.rb:58]: i would go ahead and remove the controller actions that you're not using, just to keep things clean. Having code around that you're not sure if it's used or not is a slippery slope.

File: app/views/slides/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+</p>
+
+<p>
+  <strong>Description:</strong>
+  <%= @slide.description %>
+</p>
+
+<p>
+  <strong>Img url:</strong>
+  <%= @slide.img_url %>
```
Comment (efatsi) [inline app/views/slides/show.html.erb:15]: maybe `<%= image_tag @slide.img_url %>` to get this rendering?

File: app/views/tutorial/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<h1>How to Play</h1>
+
+<% for slide in @slides %>
```
Comment (efatsi) [inline app/views/tutorial/index.html.erb:3]: damn, i didn't know you could write python in ruby!

File: spec/routing/slides_routing_spec.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+    it "routes to #update via PATCH" do
+      expect(patch: "/slides/1").to route_to("slides#update", id: "1")
+    end
+
+    it "routes to #destroy" do
+      expect(delete: "/slides/1").to route_to("slides#destroy", id: "1")
+    end
+  end
+end
```
Comment (efatsi) [inline spec/routing/slides_routing_spec.rb:38]: we normally don't bother with routing specs

File: spec/views/slides/edit.html.erb_spec.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    assert_select "form[action=?][method=?]", slide_path(@slide), "post" do
+
+      assert_select "input[name=?]", "slide[title]"
+
+      assert_select "textarea[name=?]", "slide[description]"
+
+      assert_select "input[name=?]", "slide[img_url]"
+    end
+  end
+end
```
Comment (efatsi) [inline spec/views/slides/edit.html.erb_spec.rb:24]: or view specs

File: app/models/tutorial.rb [hunk 6]
```diff
@@ -0,0 +1,3 @@
+class Tutorial < ApplicationRecord
+
+end
```
Comment (efatsi) [inline app/models/tutorial.rb:3]: I don't think you need a Tutorial model right? Unless you're thinking of having multiple different sets of Slides (beginner set and advanced maybe?)

Comment (efatsi) [reply app/models/tutorial.rb:3]: Also, looks like you don't have a db table backing this, so maybe this is just an artifact of some dev experiment that you didn't remove?

**Review by efatsi — APPROVED** [review]
Good stuff. I'd recommend cutting out any and all code that's not actively in use, but this is a pointless project so moving quickly and leaving a little wake isn't the end of the world.

File: app/views/tutorial/index.html.erb (hunk 3 above)
Comment (horrorsaur) [reply app/views/tutorial/index.html.erb:3]: ... I didn't know either. Won't lie, this one made me laugh. I just defaulted to this and it worked so I didn't think twice. I can update it to be the Ruby way.

File: app/models/tutorial.rb (hunk 6 above)
Comment (horrorsaur) [reply app/models/tutorial.rb:3]: Yeah, that's right. This was from my initial approach this morning. I'll remove it!

**Pull request merged**